.travis.yml

```diff
@@ -1,9 +1,6 @@
 language: go
 sudo: false
 go:
-  - 1.3
-  - 1.4
-  - 1.5
   - 1.7
   - 1.8
   - tip
```

Comment:
While we're here:

```
matrix:
  include:
    - go: 1.7
    - go: 1.8
    - go: 1.x
    - go: master
  allow_failures:
    - go: master
```

`1.x` will always point to the latest version. `master` points to the latest rev, which is useful for identifying breaking changes in upcoming versions.